todo_app/tests/test_endtoend.py

```diff
@@ -16,36 +16,27 @@
 load_dotenv()
 
 
-def add_board():
-    response = requests.post(
-        url=f'https://api.trello.com/1/boards',
-        params={
-            'name': 'Selenium',
-            'key': os.environ['TRELLO_KEY'],
-            'token': os.environ['TRELLO_TOKEN']
-        }
-    )
-    return response.json()['id']
+def get_board():
+    client = pymongo.MongoClient(f"mongodb+srv://{os.getenv('MONGO_USERNAME')}:{os.getenv('MONGO_PASS')}@{os.getenv('MONGO_HOST')}/?w=majority")
+    db = client[os.getenv('DEFAULT_DATABASE')]
+    os.environ['TRELLO_BOARD'] = "test_board" + uuid.uuid1().hex
+    col = os.environ['TRELLO_BOARD']
+    return db[col]
+   
 
-
-def del_board(board_id):
-    requests.delete(
-        url=f'https://api.trello.com/1/boards/{board_id}',
-        params={
-            'key': os.environ['TRELLO_KEY'],
-            'token': os.environ['TRELLO_TOKEN']
-        }
-    )
+def del_board(db):
+    db.drop()
 
 @pytest.fixture(scope='module')
 def app_with_temp_board():
     # Create the new board & update the board id environment variable
-    board_id = add_board()
-    os.environ['TRELLO_BOARD'] = board_id
-
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    db = get_board()
+    #os.environ['TRELLO_BOARD'] = "test_board"
```

Comment:
Let's delete these rather than commenting them out - easy enough to recover them with git if we need to later!